components/contextualhelp/metadata/contextualhelp.yml

```diff
@@ -18,17 +18,17 @@ examples:
               </svg>
             </button>
             <div class="spectrum-Popover is-open spectrum-Popover--bottom-start spectrum-ContextualHelp-popover" role="presentation">
-              <h2 class="spectrum-ContextualHelp-heading">Permission required</h2>
-              <p class="spectrum-ContextualHelp-body">Cupcake ipsum dolor sit amet jelly beans. Chocolate jelly caramels. Icing soufflé chupa chups donut cheesecake. Jelly-o chocolate cake sweet roll cake danish candy biscuit halvah.</p>
+              <h2 class="spectrum-Heading spectrum-Heading--sizeM spectrum-ContextualHelp-heading">Permission required</h2>
```

Comment:
I think ideally we'd simplify this such that the spectrum heading and body classes aren't needed and can be extended in the contextual help class. The more isolated a component is from other dependencies, the easier it is to convert to a web component.

Reply:
Right. I can go ahead and just use the tokens that the heading and body classes use even though they weren't specified for this component. Let me see about making that change!